lesson07/assignment/index.html

```diff
@@ -0,0 +1,131 @@
+            <a href="#" class="news main-grid__header__nav__items__item" v-on:click="makeActive('news')">News & Events</a>
+            <a href="#" class="contact main-grid__header__nav__items__item" v-on:click="makeActive('contact')">Contact</a>
+          </nav>
+        </div>
+        <script type="text/javascript">
+          const activeApp = {
+            // el: '#main',
+            data() {
+              return {
+              active: 'home'
+              }
+            },
+            methods: {
+              makeActive: function(item){
+                this.active = item;
+              }
+            }
+          }
+          Vue.createApp(activeApp).mount('#mainNavApp')
+        </script>
+      </header>
+
+      <div class="main-grid__hero">
+        <img class="main-grid__hero__img" src="./images/frustrated-mom-son-crop.jpg">
+      </div>
+      <div class="cpsn" id="app">
+        <div class="parent-info">
+          <!-- <h1>{{ parent }}</h1> -->
+          <button v-if="!imageShowing" v-on:click="toggleImage">Help, my adult son won't move out of my home!</button>
+          <button v-if="imageShowing" v-on:click="toggleImage">You need Changes! Call 1-888-000-0000</button>
+          <img v-if="imageShowing" v-bind:src="image" width="500">
+
+          <h1>One-Liners help to end conversations preventing arguments and conflict.</h1>
+          <h2>Reduce Useless Conflict:</h2>
+          <ul>
+            <li v-for="conflict in conflicts">{{ conflict }}</li>
+          </ul>
+          <h2>Define Boundaries:</h2>
+          <ul>
+            <li v-for="boundary in boundarys">{{ boundary }}</li>
+          </ul>
+          <h2>Endorse Your Child’s Capabilities:</h2>
+          <ul>
+            <li v-for="support in supports">{{ support }}</li>
+          </ul>
+        </div>
+      </div>
+      <script type="text/javascript">
+        const cpsnApp = {
+          data() {
+            return {
+              image: './images/support.jpg',
+              imageShowing: false,
+              conflicts: ['“I have said everything I am going to say about that.”', '“I will think about it.”', 'third one', '“I’m sorry you feel that way.”', '“I have nothing more to say.”', '“That’s the end of the conversation.”'],
+              boundarys: ['“No.”', '“If you need the answer right now, the answer is no."', '“That doesn’t work for me.”', '“You do not have my permission.”', '“I have decided not to.”'],
+              supports: ['“I love you, support you and know you can figure it out.”', '“Whatever you put your mind to happens.”', '“You must be very proud of yourself!”', '“You are way too independent to move back home.”', '“I think you know what the right thing to do would be.”']
+            }
+          },
+          methods: {
+            toggleImage() {
+              if (this.imageShowing === false) {
+                this.imageShowing = true
+              } else {
+                this.imageShowing = false
```

Comment:
Here's a nice way of condensing this into one line of code:
``` 
toggleImage() {
   this.imageShowing = !this.imageShowing
}
```

The `!` reverses a bool value